README.md

```diff
@@ -1,591 +1,145 @@
-# HAPI-FHIR Starter Project
+# Documentation HAPI FHIR JPA Server
 
-This project is a complete starter project you can use to deploy a FHIR server using HAPI FHIR JPA.
+## Introduction
+The purpose of this HAPI FHIR JPA Server is to provide a Proof of Concept where it functions like a metadata register. [HAPI FHIR](https://hapifhir.io/) is a complete implementation of the [HL7 FHIR](https://www.hl7.org/fhir/) standard for healthcare interoperability in Java. It is an open source software licensed under the business-friendly [Apache Software License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
Could you also state that this PoC is part of the GFModules project and explain the gfmodules project like in the coordination.

> Disclaimer
>
>The GFModules project is currently under development. The information in this repository is subject to change. The information in this repository describes the current state of the various applications.
>
>Introduction
>The GFModules project is a collection of applications that have the purpose to improve the data exchange >between healthcare providers. This project is the technical implementation of the various components of >the 'Generieke Functies, lokalisatie en addressering' project of the Ministry of Health, Welfare and Sport of >the Dutch goverment.